ulog/doc.go

```diff
@@ -21,7 +21,7 @@
 // Package ulog is the Logging package.
 //
 // package ulog provides an API wrapper around the logging library
-// zap (https://github.com/uber-go/zap). package ulog uses the builder pattern
+// zap (https://go.uber.org/zap). package ulog uses the builder pattern
```

Comment:
I like the previous link better, since godoc does'nt link back to github but github links to godoc